app/models/commit.js

```diff
@@ -43,29 +43,14 @@ export default Model.extend({
     'committerName',
     'committerEmail',
     function () {
-      let namesMatch = this.get('authorName') === this.get('committerName');
-      let emailsMatch = this.get('authorEmail') === this.get('committerEmail');
-      return namesMatch && emailsMatch;
+      if (this.get('committerEmail') !== 'noreply@github.com') {
```

Comment:
I'm not sure if this is needed. If `committerEmail` is noreply@github.com `authorEmail` could still be different and we should show both.